templates/base/package.json

```diff
@@ -40,10 +40,15 @@
   "devDependencies": {
     "@next/bundle-analyzer": "11.1.2",
     "@babel/plugin-transform-runtime": "7.15.0",
+    "@babel/preset-typescript": "^7.16.7",
```

Comment:
I think some of these dependencies must be available via Turbo Repo. We don't want to have conflict of versions an duplication.